packages/neo-one-website/tutorial/tutorial.md

```diff
@@ -68,6 +68,9 @@ npm install @neo-one/cli @neo-one/client @neo-one/smart-contract @neo-one/smart-
 
 We recommend taking a moment to [setup your editor](/docs/environment-setup#Editor-Setup) to take advantage of inline NEO•ONE compiler diagnostics.
 
+5. Review the available [configuration options](/docs/config-options) and make the necessary modifications to the `.onerc` file.
```

Comment:
What necessary modifications?  Maybe instead of the react specific note below, you just say something like "Here are the config options used in this tutorial." and then give the example config with the correct options?

Take or leave this comment, just think maybe we could be a little clearer here.

Reply:
How's this... 
https://github.com/neo-one-suite/neo-one/pull/1177/files#diff-441447123cbc184d48bd6602a2c76df1R72